tests/mongodb/collection/source/app.d

```diff
@@ -0,0 +1,32 @@
+/// Requires mongo service running on localhost with default port
+/// Uses test database
+
+module app;
+
+import vibe.core.core;
+import vibe.core.log;
+import vibe.data.bson;
+import vibe.db.mongo.mongo;
+
+import std.algorithm : all, canFind, equal, map, sort;
+import std.conv : to;
+import std.encoding : sanitize;
+import std.exception : assertThrown;
+
+
+void runTest(ushort port)
+{
+	MongoClient client = connectMongoDB("127.0.0.1", port);
+
+	/// Drop a collection
```

Comment:
is this test only testing dropping non-existent collections?

Should probably drop it twice just in case if it randomly existed beforehand on CI. It should behave the same both times after all.

Otherwise it might make sense to also create a collection here.